### DISCUSSION TITLE
Override backup params

### DISCUSSION THREAD

**Discussion by 2020testuser** [vmware-tanzu/velero#3331] [Community support Q&A] [answered]
**Describe the problem/challenge you have**
Hi,
Could you please let me know whether I can override the s3 url and bucket name when I execute the application namespace backup from my local machine?

I'm in application team. I have velero CLI installed in my local machine. In on-prem Kubernetes Cluster, the admin has installed the Velero Server.  I want to override the default s3 url/bucket when I take my application namespace backup from my local machine. 

**Describe the solution you'd like**
Allow application teams to take their own backups of their application backups from their local machine by overriding the s3 url and bucket name (by passing these in velero backup command).

 

**Environment:**

- Velero version (use `velero version`): 1.5.2
- Kubernetes version (use `kubectl version`):  OnPrem Kubernetes with 1.16.x

Thanks in advance!

**Comment by zubron** [comment, marked as answer]
Hi @2020testuser. In order to use a different S3 bucket, you will need to configure a different [Backup Storage Location](https://velero.io/docs/v1.5/locations/). There is also a [current limitation](https://github.com/vmware-tanzu/velero/issues/3304) that Velero can only support one set of credentials per provider so the current credentials that are being used for the current S3 bucket will also need to be valid for any other S3 buckets.

If you are able to configure a different location, then you can specify which location to use when creating a backup using the `--storage-location` flag. See `velero backup create --help` for more details.
